app/controllers/orders/placed.js

```diff
@@ -18,17 +18,40 @@ export default Controller.extend({
   }),
 
   actions: {
-    processStripeToken() {
+    processStripeToken(token) {
       // Send this token to server to process payment
+      let order = this.get('model');
+      let chargePayload = {
+        'data': {
+          'attributes': {
+            'stripe'            : token.id,
+            'paypal_payer_id'   : null,
+            'paypal_payment_id' : null
+          },
+          'type': 'charge'
+        }
+      };
+      let config = {
+        skipDataTransform: true
+      };
+      chargePayload = JSON.stringify(chargePayload);
+      return this.get('loader').post(`orders/${order.identifier}/charge`, chargePayload, config)
+        .then(charge => {
+          if (charge.data.attributes.status) {
+            this.get('notify').success('Payment was done successfully.');
+            this.transitionToRoute('orders.view', order.identifier);
+          } else {
+            this.get('notify').error('Payment was not successful. Please try again or contact your card customer support.');
```

Comment:
The server returns the error message received from Stripe to the FE. Please show that message so that they know what went wrong exactly.

Reply:
@srv-twry done. Please check.